src/qtiCommonRenderer/renderers/interactions/TextEntryInteraction.js

```diff
@@ -132,7 +132,7 @@ var render = function render(interaction) {
 
             hideTooltip($input);
 
-            if ($input.val().length && regex.test($input.val())) {
+            if ($input.val().length && regex.test($input.val()) || $input.val().length === 0 && regex.test($input.val())) {
```

Comment:
OK, now this is the same as writing
```suggestion
            if (regex.test($input.val())) {
```

Reply:
But I still strongly suggest to re-check that failing test, and understand if it actually checks the right behavior.